BubbleSort/main.cpp

```diff
@@ -7,6 +7,7 @@
 //
 
 #include <iostream>
+#include <stdlib.h> //for rand() function
```

Comment:
I can't see rand() method usage. If this is unnecessary, please remove it.

Reply:
https://github.com/muhammedkarakul/BubbleSortAlgorithm/blob/037712478c4b3313c86402eda66422efe33b2fd8/BubbleSort/main.cpp#L38

Reply:
https://github.com/muhammedkarakul/BubbleSortAlgorithm/blob/037712478c4b3313c86402eda66422efe33b2fd8/BubbleSort/main.cpp#L40-L46

Reply:
@muhammedkarakul  rand() function is used in Main.

Reply:
Ok sorry, my mistake. I was don't check full main.cpp code 👍